sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -489,4 +491,12 @@ public static void setFeedOptionsMaxItemCount(FeedOptions feedOptions, Integer m
     public static <T> CosmosContinuablePagedFlux<T> createCosmosContinuablePagedFlux(Function<CosmosPagedFluxOptions, Flux<FeedResponse<T>>> pagedFluxOptionsFluxFunction) {
         return new CosmosContinuablePagedFlux<>(pagedFluxOptionsFluxFunction);
     }
+
+    public static Mono<FeedResponse<Document>> readMany(CosmosAsyncContainer container, 
```

Comment:
Is a sync version of this contract also needed?

Reply:
At the moment, I added async version, sync wrapper can be added quickly if required.

Reply:
we added sync version too.

Reply:
Thanks Mo for adding sync API. We have both sync and async API now.